src/org/labkey/test/util/AuditLogHelper.java

```diff
@@ -69,6 +76,116 @@ public DataRegionTable goToAuditEventView(String eventType)
         return new DataRegionTable("query", _wrapper);
     }
 
+    /**
+     * Get the audit logs from the LabKey server filtered to the given project.
+     *
+     * @param projectName Name of the LK project to filter to.
+     * @param folderName Name of the LK folder to filter to.
+     * @param auditEventName Name of the audit event to filter on. Example 'SamplesWorkflowAuditEvent'.
+     * @param columnNames The name of the columns to return.
+     * @param filters The filters to be applied
+     * @return A rowResponse with the query logs.
+     * @throws IOException Can be thrown by the SelectRowsCommand.
+     * @throws CommandException Can be thrown by the SelectRowsCommand.
+     */
+    public static SelectRowsResponse getAuditLogsFromLKS(String projectName, String folderName, String auditEventName, List<String> columnNames,
+                                                         List<Filter> filters, @Nullable Integer maxRows) throws IOException, CommandException
+    {
+        SelectRowsCommand cmd = new SelectRowsCommand("auditLog", auditEventName);
+        cmd.setColumns(columnNames);
+        cmd.addFilter("ProjectId/DisplayName", projectName, Filter.Operator.EQUAL);
```

Comment:
This filter should use the project path, not the display name. We don't, generally, set custom container display names but we might some day.
```suggestion
        cmd.addFilter("ProjectId/Path", projectName, Filter.Operator.EQUAL);
```

Reply:
I agree. I was just moving this method over from SMTestUtils, but I'll make this change.